settings.py

```diff
@@ -15,6 +15,10 @@
     }
 }
 
+TIMEZONE = "Ukraine/Kiev"
 USE_TZ = False
```

Comment:
Consider setting `USE_TZ = True` to enable timezone-aware datetimes. This is generally recommended for Django projects to handle timezones correctly.